doc/examples/applications/plot_human_mitosis.py

```diff
@@ -76,10 +76,10 @@
 fig, ax = plt.subplots(ncols=2, figsize=(10, 5))
 ax[0].imshow(image)
 ax[0].set_title('Original')
-ax[0].axis('off')
+ax[0].set_axis_off()
```

Comment:
I'd like to ask the same question @lagru did in https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1527178981. Maybe because `set_axis_off()` looks more like `set_title()` and the other `set_` functions (so, visually and mentally, it's nicer/lighter/smoother)? @GParolini I'd be curious if you had other ideas on this!

Reply:
Both axis("off") and set_axis_off() were used. I always changed axis("off") in set_axis_off() because all the other axis methods (title, xlim, etc.) use set. It seemed  to me more natural to have the same structure for everything.